wpilibc/src/main/native/include/frc/DriverStation.h

```diff
@@ -46,6 +49,60 @@ class DriverStation final {
     kElimination
   };
 
+  /**
+   * A controller POV direction.
+   */
+  enum POVDirection : uint8_t {
+    /// POV center.
+    kCenter = HAL_JoystickPOV_kCentered,
+    /// POV up.
+    kUp = HAL_JoystickPOV_kUp,
+    /// POV up right.
+    kUpRight = HAL_JoystickPOV_kRightUp,
+    /// POV right.
+    kRight = HAL_JoystickPOV_kRight,
+    /// POV down right.
+    kDownRight = HAL_JoystickPOV_kRightDown,
+    /// POV down.
+    kDown = HAL_JoystickPOV_kDown,
+    /// POV down left.
+    kDownLeft = HAL_JoystickPOV_kLeftDown,
+    /// POV left.
+    kLeft = HAL_JoystickPOV_kLeft,
+    /// POV up left.
+    kUpLeft = HAL_JoystickPOV_kLeftUp,
+  };
+
+  /**
+   * Gets the angle of a POVDirection.
+   *
+   * @param angle The POVDirection to convert.
+   * @return The angle clockwise from straight up, or std::nullopt if the
+   * POVDirection is kCenter.
+   */
+  static constexpr std::optional<Rotation2d> GetAngle(POVDirection angle) {
+    switch (angle) {
+      case kCenter:
```

Comment:
`default` instead of `case kCenter`. In C++, this actually won't compile as not all cases return a value.